ait/core/server/client.py

```diff
@@ -37,11 +37,13 @@ def __init__(
         # calls gevent.Greenlet or gs.DatagramServer __init__
         super(ZMQClient, self).__init__(**kwargs)
 
-    def publish(self, msg):
+    def publish(self, msg, topic=None):
         """
         Publishes input message with client name as topic.
```

Comment:
Request a minor tweak to doc to indicate topic can be specified, or client name used as default